lib/src/main/java/uk/gov/di/ipv/cri/experian/kbv/api/gateway/StartAuthnAttemptRequestMapper.java

```diff
@@ -40,27 +53,31 @@ private void setResidency(SAARequest saaRequest) {
         saaRequest.getResidency().add(residency);
     }
 
-    private void setApplicationData(SAARequest saaRequest) {
+    private void setApplicationData(SAARequest saaRequest, String strategy) {
         ApplicationData applicationData = new ApplicationData();
         applicationData.setApplicationType("IG");
         applicationData.setChannel("IN");
         applicationData.setSearchConsent("Y");
-        applicationData.setProduct("3 out of 4");
+        applicationData.setProduct(StringUtils.isNotBlank(strategy) ? strategy : "3 out of 4");
```

Comment:
it's probably worth declaring a variable entitled defaultStrategy to help explain the "3 out of 4"